tests/maybe.test.ts

```diff
@@ -202,3 +202,16 @@ test('eq - some is `eq` to some if the contents are ===', () => {
     // Not same object, not ====
     expect(some(x).eq(some({}))).toBe(false);
 });
+
+// -------
+// Fantasy
+// -------
+
+test('fantasy-land/map - calls into the map method', () => {
```

Comment:
This doesn't really test much in terms of functionality, that's covered by `map` above. This just tests that the alias works the way I hoped it would. (it does!)